pegasus/sites.v3/code.org/public/accept-terms.haml

```diff
@@ -0,0 +1,50 @@
+---
+title: Updated Terms of Service and Privacy Policy
+layout: page_print
+---
+
+%link{:href=>"/css/accept-terms-interstitial.css", :rel=>"stylesheet"}
+%script{:src=>"/js/accept-terms.js"}
+
+.container
+  %h2 Updated Terms of Service and Privacy Policy
+
+  We updated our Terms of Service and Privacy Policy, as shown below. The changes are effective as of August 22, 2016.  The biggest change: we will
+  %i no longer store email addresses for student accounts on Code Studio.
+  This is a big deal for us, and we hope it sets a new standard for student privacy in education technology. We’ve sent you a summary of the other changes via email.
+
+  %br
+  %br
+  You’ll need to accept these new terms for students under the age of 13 in your classroom to have full access to all the latest features in Code Studio.  When registering an account for a student under 13, you will be asked to confirm that your school has parental consent for the collection of the student’s personal information for the use and benefit of the school, including educational programs such as Code Studio.
+
+  %br
+  %br
+  .scroll-box
+    %h1.no-top-margin Terms of Service
+    = view :tos
+    %hr
+    %h1 Privacy Policy
+    = view :privacy
+
+  .left
+    %form#accept-terms-form{role: 'form', onsubmit: 'event.preventDefault();'}
+      .form-group
+        .checkbox
+          %label
+            %input#accept-terms-checkbox{name: 'accept_terms_b', type: 'checkbox', value: '1'}/
+            I agree to the updated
```

Comment:
Does this make the entire sentence clickable to highlight the checkbox?  (That would be ideal.)